server/appengine/src/main/java/who/WhoServiceImpl.java

```diff
@@ -60,6 +62,28 @@ public Void putLocation(PutLocationRequest request) throws IOException {
     return new Void();
   }
 
+  @Override
+  public Void putNotificationSettings(PutNotificationSettingsRequest request)
+    throws IOException {
+    // TODO: Consider doing this in some form of datastore transaction. THe trick being that the firebase and datastore may get out of sync...
+    Client client = Client.current();
+
+    // The underlying "Topics" system allows for a null country code; would that be OK to allow here too? ?????????????????????????????????????????????
```

Comment:
`ZZ` is a valid alpha2 code that's user assigned, so we could use that:
https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Decoding_table

Reply:
The system uses null/"" as "country not set" elsewhere on client and server, which seems right to me. The question here is a design question: what is the intent of this API.